.github/workflows/merge-tests.yml

```diff
@@ -96,16 +96,22 @@ jobs:
           path: master
       # Copy the script so the base can bench with the same parameters
       - name: Copy source script to base branch
-        run: cd benches && cp justfile bench.env ../master/benches
+        run: cp justfile bench.env ../master/benches
+        working-directory: benches
       - name: Set base ref variable
-        run: cd master && echo "BASE_REF=$(git rev-parse HEAD)" >> $GITHUB_ENV
-      - run: echo ${{ env.BASE_REF }}
+        run: echo "BASE_REF=$(git rev-parse HEAD)" | tee -a $GITHUB_ENV
+        working-directory: master
       - name: Run GPU bench on base branch
-        run: cd master/benches && just --dotenv-filename bench.env gpu-bench fibonacci_lem
+        run: just --dotenv-filename bench.env gpu-bench fibonacci_lem
+        working-directory: master/benches
       - name: Copy bench output to PR branch
-        run: cp master/${{ env.BASE_REF }}.json .
+        run: |
+          mkdir -p target
+          cp -r master/target/criterion target
+          cp master/${{ env.BASE_REF }}.json .
       - name: Run GPU bench on PR branch
-        run: cd benches && just --dotenv-filename bench.env gpu-bench fibonacci_lem
+        run: just --dotenv-filename bench.env gpu-bench fibonacci_lem
+        working-directory: benches
```

Comment:
I never remember if the working directory attribute is relative to the one in the prior step, so I tend to write my working-directories in absolute form, relative to the `${{ github.workspace }}` root (that is `working-directory: ${{ github.workspace }}/path/to/dir`). But YMMV.

Reply:
Thanks, any precautions in this slow edit-debug cycle are much appreciated!